AUTHORS

```diff
@@ -31,6 +31,7 @@ The following people have provided patches or other contributions:
   DD Liu
   Betsy Riley
   Robert Jacobs
+  Google Inc.
```

Comment:
They claim copyright on the code in the patch, and the AUTHORS file is, among other things, the list of copyright holders.